packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
```

Comment:
I got tired of copy-pasting between parse-cjs and parse-pre-cjs and disabling archive tests every time I work on the logic.

Reply:
Fantastic.